fastcrypto-vdf/src/math/hash_prime.rs

```diff
@@ -110,16 +71,38 @@ pub fn hash_prime_default(seed: &[u8], length_in_bytes: usize, bitmask: &[usize]
 pub struct DefaultPrimalityCheck {}
 
 impl PrimalityCheck for DefaultPrimalityCheck {
-    fn is_prime(x: &BigUint) -> bool {
-        is_prime(x, None).probably()
+    fn is_probable_prime(x: &BigUint) -> bool {
+        let rounds = compute_number_of_rounds(x.bits());
+        let mut config = PrimalityTestConfig::default();
+        config.sprp_trials = 0;
+        config.sprp_random_trials = rounds as usize;
+        is_prime(x, Some(config)).probably()
+    }
+}
+
+fn compute_number_of_rounds(bits: u64) -> u64 {
+    // Assuming that the input is a random odd number, we get the following estimates Damgård,
```

Comment:
is there a NIST recommendation on this?

Reply:
Maybe. I know that there is an ISO recommendation which we could follow.